### ISSUE TITLE
Multiple (almost) identical pull requests

### ISSUE THREAD

**Issue by nicolasfella** [flathub-infra/flatpak-external-data-checker#322]
Today https://github.com/flathub/org.kde.itinerary got 6 pull requests containing almost the same changes (sometimes 7 modules are updated, sometimes 8)

There should only be one PR containing all changes, or multiple with one updated module each

**Comment by gasinvein** [comment]
Looks like the same problem as #315 - version fluctuations across `download.kde.org` mirrors.

**Comment by johanfleury** [comment]
Same issue on [flathub/com.jetbrains.PyCharm-Professional](https://github.com/flathub/com.jetbrains.PyCharm-Professional/pulls). I’m not even reading those PRs anymore.

Honestly I’m wondering if this external-data-checker is the correct approach. There are many great FLOSS tools that do version checking (renovate is a good example), why not build upon it?

**Comment by gasinvein** [comment]
> I’m not even reading those PRs anymore.

Why then even configure `x-checker-data` for dozens of dependencies if you never intended to review and/or update at least some of them?

**Comment by johanfleury** [comment]
It was a contributor who did that (I’m barely maintaining this package) and I believe having automatic updates is a good think. I just don’t think I should receive 3 PR a day for the exact same thing.

**Comment by gasinvein** [comment]
tl; dr remove `x-checker-data` entries from PyPI sources.

>I just don’t think I should receive 3 PR a day for the exact same thing.

It's not the same thing. Each PR has slightly different packages updated to different versions. If you've merged e.g. [this one](https://github.com/flathub/com.jetbrains.PyCharm-Professional/pull/76) a month ago, then the set of updated packages would be much smaller and the difference between changes would be more clear.

>I believe having automatic updates is a good think

Flatpak-builder manifests don't have any kind of technical relation between source items whatsoever, and f-e-d-c can't can ensure consistent dependencies and buildable updates for you, not in a universal way at least. So no, dumping lots of `x-checker-data` stanzas to almost every source is not a good idea at all.

>It was a contributor who did that

And they didn't do any favour to neither you nor me.
